### PR TITLE
Updates android-browser-helper and enables WebView fallback on config

### PR DESCRIPTION
**Pull request by andreban**
- Updates to using android-browser-helper-1.2.0
- Adds an option to enable the WebView fallback on the twa-manifest.json

### CODE REVIEW

File: packages/core/src/spec/lib/TwaManifestSpec.ts [hunk 1]
```diff
@@ -72,6 +72,7 @@ describe('TwaManifest', () => {
       expect(twaManifest.generateShortcuts())
           .toBe('[[name:\'shortcut name\', short_name:\'short\',' +
             ' url:\'https://pwa-directory.com/launch\', icon:\'shortcut_0\']]');
+      expect(twaManifest.fallbackType).toBe('customtabs');
```
Comment (PEConn) [inline packages/core/src/spec/lib/TwaManifestSpec.ts]: Can we add a test as well to check that `webview` works?

File: packages/core/template_project/app/build.gradle [hunk 2]
```diff
@@ -36,7 +36,10 @@ def twaManifest = [
     shortcuts: <%= generateShortcuts() %>,
     // The duration of fade out animation in milliseconds to be played when removing splash screen.
     splashScreenFadeOutDuration: <%= splashScreenFadeOutDuration %>,
-    generatorApp: '<%= generatorApp %>' // Apllication that generated the Android Project
+    generatorApp: '<%= generatorApp %>', // Apllication that generated the Android Project
```
Comment (PEConn) [inline packages/core/template_project/app/build.gradle]: nit: type in "Apllication"

File: packages/core/template_project/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -21,6 +21,7 @@
      the abbreviated format. -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="<%= packageId %>">
+    <uses-permission android:name="android.permission.INTERNET"/>
```
Comment (PEConn) [inline packages/core/template_project/app/src/main/AndroidManifest.xml]: Did this get added in by Android Studio or do we need it for WebView? It's seems a little ugly including it even if we don't need WebView, but on the other hand it's not that much overhead.

Comment (andreban) [reply packages/core/template_project/app/src/main/AndroidManifest.xml]: This needs to be added for the WebView to work on Android 9 and below. We conditionally add it, only when the fallback is type `webview`

Comment (andreban) [reply packages/core/template_project/app/src/main/AndroidManifest.xml]: Missed the "could" on the text. Fixed that.

Comment (PEConn) [reply packages/core/template_project/app/src/main/AndroidManifest.xml]: If it's easy enough let's do it, I don't think it's blocking though.

Comment (andreban) [reply packages/core/template_project/app/src/main/AndroidManifest.xml]: Done

File: packages/core/template_project/app/build.gradle (hunk 2 above)
Comment (andreban) [reply packages/core/template_project/app/build.gradle]: done

File: packages/core/src/spec/lib/TwaManifestSpec.ts (hunk 1 above)
Comment (andreban) [reply packages/core/src/spec/lib/TwaManifestSpec.ts]: done

**Comment by andreban** [comment]
Closes #105

**Pull request merged**